blivet/devicetree.py

```diff
@@ -144,6 +142,14 @@ def devices(self):
 
         return devices
 
+    @property
+    def names(self):
+        """ List of devices names """
+        lv_info = list(lvs_info.cache.keys())
+        names = [dev.name for dev in self.devices] + self._hidden
```

Comment:
This should use `self._devices` (private/hidden) instead of `self.devices` (public) so it includes any incomplete devices that are present.

Reply:
Fixed.